addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -965,7 +965,7 @@ export function isUnremovable(node) {
     const isEditableRoot =
         node.isContentEditable &&
         node.parentElement &&
-        !node.parentElement.isContentEditable &&
+        !(node.parentElement.isContentEditable && node.nodeName !== 'A') &&
```

Comment:
This can't possible fix the problem : it does nothing ^^

`!(a && !b) && !b` == `(!a || b) && !b`
so if `a`, then `b && !b` must be true, which is obviously impossible :)

Reply:
maybe `node.parentElement.nodeName` ?